beeline/middleware/django/test_django.py

```diff
@@ -18,6 +18,7 @@ def test_call_middleware(self):
         ''' Just call the middleware and ensure that the code runs '''
         mock_req = Mock()
         mock_resp = Mock()
+        mock_resp.return_value.streaming = False
```

Comment:
Funny enough, when we were having our discussion in Slack, on of the first things I tried to do was crack open the tests and see if there was a way to exercise the case of a streaming response. I didn't get very far (I'm not that familiar with the mechanics of Django), but I think it's definitely the sort of thing we _should_ have tests around. Any ideas?

Reply:
You are absolutely right with the nudge that the above code deserves a test. :)

I'll see if I have time to whip one up this evening.